jobbergate-docs/src/conf.py

```diff
@@ -31,6 +31,7 @@
 exclude_patterns = []
 extensions = [
     "sphinx.ext.githubpages",
+    "sphinx.ext.autosectionlabel",
```

Comment:
What does this do?

Reply:
This is for cross-references, like this one:

https://github.com/omnivector-solutions/jobbergate/blob/df3660a4e0aa21cc7ade37a6f3ec652501b1b1ea/jobbergate-docs/src/developer_guide/ci.rst#L15